### PR TITLE
Linked cases

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5565
https://tools.hmcts.net/jira/browse/EUI-5298

### Change description ###
Link cases and unlink cases

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: api/configuration/appConfig.ts [hunk 1]
```diff
@@ -73,6 +74,7 @@ export const applicationConfiguration = () => {
     servicesCcdCaseAssignmentPath: getConfigValue(SERVICES_CCD_CASE_ASSIGNMENT_API_PATH),
     servicesCcdComponentPath: getConfigValue(SERVICES_CCD_COMPONENT_API_PATH),
     servicesCcdDataStorePath: getConfigValue(SERVICES_CCD_DATA_STORE_API_PATH),
+    servicesCommonrdApiPath: getConfigValue(SERVICES_COMMONRD_API_URL),
```
Comment (rayliangatos) [inline api/configuration/appConfig.ts:77]: can be removed this is duplicated

File: api/configuration/references.ts [hunk 2]
```diff
@@ -63,6 +63,8 @@ export const SERVICES_PRD_LOCATION_API = 'services.prd.locationApi';
 export const SERVICES_PRD_JUDICIAL_API = 'services.prd.judicialApi';
 export const SERVICES_PRD_COMMONDATA_API = 'services.prd.commondataApi';
 
+export const SERVICES_COMMONRD_API_URL = 'services.commonrd.api';
```
Comment (rayliangatos) [inline api/configuration/references.ts:66]: can be removed this is duplicated with SERVICES_PRD_COMMONDATA_API,

File: config/custom-environment-variables.json [hunk 3]
```diff
@@ -87,7 +87,10 @@
     "payments": "SERVICES_PAYMENTS_URL",
     "s2s": "SERVICES_S2S",
     "termsAndConditions": "SERVICES_TERMS_AND_CONDITIONS",
-    "payment_return_url": "SERVICES_PAYMENT_RETURN_URL"
+    "payment_return_url": "SERVICES_PAYMENT_RETURN_URL",
+    "commonrd": {
+      "api": "SERVICES_COMMONRD_API"
+    }
```
Comment (rayliangatos) [inline config/custom-environment-variables.json:93]: can be removed this is duplicated with SERVICES_PRD_COMMONDATA_API,

File: api/.env.defaults [hunk 4]
```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```
Comment (rayliangatos) [inline api/.env.defaults:3]: could this be reverted?

**Comment by johnbenjamin-hmcts** [comment]
The changes have been pulled into PR 2507 and this PR is no longer required.

**Pull request closed without merging**